### DISCUSSION TITLE
Thread Limits and Staging Drives

### DISCUSSION THREAD

**Discussion by stikeleather** [ericaltendorf/plotman#504] [Ideas]
I think there should be a parameter for global threads limit. 

I'd also like a staging drive (intermediate storage) implementation that will check for plots on the staging drive, then move to destination storage.

**Comment by Keppto** [comment]
I saw that staging drives can improve performance by dumping from the tmp drive to another slower but high capacity ssd, rather than having to wait for the write from the temp to the slow hard drive to finish. I'd also like to see this implemented into plotman.
